docs/content/concepts/ops-jobs-graphs/jobs.mdx

```diff
@@ -1,283 +1,71 @@
-
-
-class DoSomethingConfig(Config):
-    config_param: str
-
-
-@op
-def do_something(context, config: DoSomethingConfig) -> None:
-    context.log.info("config_param: " + config.config_param)
-
-
-class SimplifiedConfig(Config):
-    simplified_param: str
-
-
-@config_mapping
-def simplified_config(val: SimplifiedConfig) -> RunConfig:
-    return RunConfig(
-        ops={"do_something": DoSomethingConfig(config_param=val.simplified_param)}
-    )
-
-
-@job(config=simplified_config)
-def do_it_all_with_simplified_config():
-    do_something()
-
-
-if __name__ == "__main__":
-    # Will log "config_param: stuff"
-    do_it_all_with_simplified_config.execute_in_process(
-        run_config={"simplified_param": "stuff"}
-    )
-```
+- **Automation**: With [schedules](/concepts/partitions-schedules-sensors/schedules) and [sensors](/concepts/partitions-schedules-sensors/sensors), jobs can be used to automate the execution of your Dagster pipelines. Refer to the [Automating your data pipelines guide](/guides/dagster/automating-pipelines) for more info and examples.
+- **Control job run priority**: If using a [run queue](/deployment/run-coordinator), you can apply custom prioritization rules to how job runs are prioritized and executed.
+- **Potential for improved efficency**: By applying concurrency limits to job runs, there may be benefits to your pipeline's efficiency. Refer to the [Limiting run concurrency guide](/guides/limiting-concurrency-in-data-pipelines) for more info and examples.
 
 ---
 
-## Making jobs available to Dagster tools
-
-You make jobs available to the UI, GraphQL, and the command line by including them in <PyObject object="Definitions"/> object at the top-level of Python module or file. The tool loads that module as a code location. If you include schedules or sensors, the code location will automatically include jobs that those schedules or sensors target.
-
-```python file=/concepts/ops_jobs_graphs/repo_with_job.py
-from dagster import Definitions, job
-
-
-@job
-def do_it_all():
-    ...
+## Uses
 
+With jobs, you can:
 
-defs = Definitions(jobs=[do_it_all])
-```
+- Automate the execution of [Software-defined Assets](/concepts/assets/software-defined-assets) and [ops](/concepts/ops-jobs-graphs/ops)
+- Materialize a selection of assets based on a schedule or external trigger (sensor)
+- Apply metadata about jobs and runs using [tags](/concepts/ops-jobs-graphs/metadata-tags)
+- View and launch runs of jobs in the [Dagster UI](/concepts/webserver/ui)
 
 ---
 
-## Testing jobs
+## Implementation
 
-Dagster has built-in support for testing, including separating business logic from environments and setting explicit expectations on uncontrollable inputs. Refer to the [Testing guide](/concepts/testing) for more info and examples.
+Jobs are supported for both Software-defined Assets and ops, but the implementation for each concept is unique. Refer to the following documentation for more info:
```

Comment:
similar feedback as mentioned [here](https://github.com/dagster-io/dagster/pull/16511#discussion_r1337802711). could we merge this into the "Uses" section?

Reply:
Done!